spec/models/state_file_az_intake_spec.rb

```diff
@@ -496,45 +496,88 @@
              df_data_imported_at: 2.minutes.ago,
              message_tracker: { "messages.state_file.finish_return" => (Time.now - 2.hours).utc.to_s }
     }
+    let!(:az_intake_received_reminder_not_recent) {
+      create :state_file_az_intake, email_address: "test@example.com",
+             email_address_verified_at: 1.hour.ago,
+             email_notification_opt_in: 1,
+             df_data_imported_at: 2.minutes.ago,
+             message_tracker: { "messages.state_file.finish_return" => (Time.now - 28.hours).utc.to_s }
+    }
+    let!(:az_intake_has_disqualifying_df_data) {
+      create :state_file_az_intake,
+             filing_status: :married_filing_separately,
+             email_address: "test@example.com",
+             email_address_verified_at: 1.hour.ago,
+             email_notification_opt_in: 1,
+             df_data_imported_at: 2.minutes.ago
+    }
+    let!(:az_intake_received_reminder_not_recent_disqualifying_df_data) {
+      create :state_file_az_intake, email_address: "test@example.com",
+             filing_status: :married_filing_separately,
+             email_address_verified_at: 1.hour.ago,
+             email_notification_opt_in: 1,
+             df_data_imported_at: 2.minutes.ago,
+             message_tracker: { "messages.state_file.finish_return" => (Time.now - 28.hours).utc.to_s }
+    }
+    let!(:az_intake_has_submitted_duplicate) {
+      create :state_file_az_intake,
+             email_address: "test@example.com",
+             email_address_verified_at: 1.hour.ago,
+             email_notification_opt_in: 1,
+             df_data_imported_at: 2.minutes.ago,
+             hashed_ssn: "111443333"
+    }
+    let!(:az_intake_submitted_duplicate) {
+      create :state_file_az_intake,
+             email_address: "test@example.com",
+             email_address_verified_at: 1.hour.ago,
+             email_notification_opt_in: 1,
+             phone_number: nil,
+             df_data_imported_at: 2.minutes.ago,
+             hashed_ssn: "111443333"
+    }
```

Comment:
Removing one of the duplicate intakes, since I don't understand the need for both `az_intake_has_submitted_duplicate` and `az_intake_submitted_duplicate`